kolibri/plugins/learn/assets/src/state/actions/recommended.js

```diff
@@ -76,10 +68,10 @@ function _showRecSubpage(store, getContentPromise, pageName, windowTitleId, chan
     getContentPromise(state, channelId),
     setAndCheckChannels(store),
     // resolves to mapped content set because then resolves to its function's return value
-  ]).then(([content]) => _mapContentSet(content), error => error);
+  ]).then(([content, channels]) => [_mapContentSet(content), error => error, channels]);
```

Comment:
The scope of the second array is wrong here, the error handling function should be outside of the array, and channels should be the second element.

Reply:
Gotcha. I think the `error => error` can be dropped, since it defaults to identity function.

Reply:
Amazingly, the bad array didn't seem to have an effect on the code, though. Apparently, a function has length `0`, and there is nothing in the pagePrep.then callback utilizing channels if length is not 0.

Reply:
I would guess it is `undefined` rather than zero, but still falsy nevertheless!